.github/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+## Motivation and Context
+
+<!--
+  Why is this change required? What problem does it solve?
```

Comment:
Rather than these being comments, I think having them be normal markdown text is the standard approach for templates?

Reply:
I see the comments as a way to avoid writer's block and get the author typing. Unless there's a good way to get started, I'd prefer to leave them as comments.